GitUI/CommandsDialogs/SettingsDialog/SettingsTreeViewUserControl.cs

```diff
@@ -152,6 +152,10 @@ private void textBoxFind_TextChanged(object sender, EventArgs e)
             foreach (var node in _nodesFoundByTextBox)
             {
                 HighlightNode(node, true);
+                for (var parent = node.Parent; parent != null; parent = parent.Parent)
```

Comment:
why can't we just call `node.EnsureVisible();` instead of looping through?

Reply:
@RussKie, because I did not find it since the name has nothing to do with "expand".
It works well.